src/urllib3/util/retry.py

```diff
@@ -289,6 +297,8 @@ def get_backoff_time(self) -> float:
             return 0
 
         backoff_value = self.backoff_factor * (2 ** (consecutive_errors_len - 1))
+        if self.backoff_jitter > 0.0:
```

Comment:
Tenacity seems to allow a negative jitter implicitly. Is there a reason to disallow it here?
https://github.com/jd/tenacity/blob/433324956abb028f6d993195d31e4dd8308115c3/tenacity/wait.py#L222
```
>>> import random
>>> random.uniform(0, -1)
-0.45467571802820805
```

Reply:
Interesting. We can leave that in here too, sure, but I'll continue to leave out the random.random call in the most common (default) 0.0 case?

Reply:
I've added that in with a minimum of zero

Reply:
> but I'll continue to leave out the random.random call in the most common (default) 0.0 case?

Good, this is an efficient way